docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -108,6 +108,8 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 ## Markdown (`markdown`)
 
 - `ENABLE_HARD_LINE_BREAK`: **false**: Enable Markdown's hard line break extension.
+- `CUSTOM_URL_SCHEMES`: : Use a comma separated list (http,https,ftp,git,svn) to indicate 
+  URI hyperlinks that should be rendered in Markdown. The default is only http and https.
```

Comment:
It's not URI but URL in this case. Also, http and https are always rendered no matter what this setting says.

Reply:
I think this should be part of the pr that does the custom URL schemes assuming we haven't merged that yet

Reply:
@guillep2k @zeripath Thanks for suggestions. Updated contents as suggested. Feel free to incorporate into pr for custom URL schemes.